robocop/checkers/tags.py

```diff
@@ -172,7 +174,9 @@ class TagNameChecker(VisitorChecker):
         "robot:recursive-continue-on-failure",
         "robot:skip",
         "robot:skip-on-failure",
+        "robot:stop-on-failure",
         "robot:exclude",
+        "robot:private",
```

Comment:
Shouldn't we test against all of those reserved tags?

Reply:
I've decided to go with equivalence class theory here - we're testing for most of them and we're using this set in our code, so we don't need to cover all possible options.